projects/tiempo-manu/tiempo-app/src/config.js

```diff
@@ -0,0 +1,8 @@
+let apiRoot = "https://api.openweathermap.org/data/2.5"
+let apiKey = "6da0a6832667e16acc2f2c49e8f1575c"
```

Comment:
Aunque sea una api key de demo casi utilizaría un .env de vue y no pondría la API key aquí.

https://cli.vuejs.org/guide/mode-and-env.html#environment-variables